webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require("path");
```

Comment:
I am going to modify the config. Example: aliases, more info about output path and etc things.

Reply:
By the way, i will use babel in order to transpile our code and to use ES6+ features.